src/volt/components/ProposalsList/SingleProposal/styles.js

```diff
@@ -44,6 +44,10 @@ export const Topic = styled(Flex).attrs(() => ({
   borderColor: "voltBrandDarkGrey"
 }))`
   border: 1px solid;
+
```

Comment:
```suggestion
margin-right: 5px;
&:last-of-type{
  margin-right: 0;
}
```

Reply:
Simply to remove technical debt and be consistent with other parts, where we define bottom and right margins on items.